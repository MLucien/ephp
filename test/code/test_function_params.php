<?php

function data($who) {
	print "Hello $who!\n";
	return "--$who--";
}

print data("World") . 'laksjdf\n';

