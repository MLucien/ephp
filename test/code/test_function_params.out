Hello World!
--World--laksjdf\n